src/window.rs

```diff
@@ -218,20 +221,18 @@ impl WindowWrapper {
 
     pub fn handle_pointer_motion(&mut self, x: i32, y: i32) {
         let previous_position = self.mouse_position;
-        if let Ok(new_mouse_position) = LogicalSize::from_physical_size_tuple(
```

Comment:
Again, mouse positions are handled in grid coordinates. So this conversion is still necessary.

Reply:
Oh I see. You are. My bad